doc/dash/dash-sonic-hld.md

```diff
@@ -482,6 +491,7 @@ DASH_ROUTING_APPLIANCE_TABLE:{{appliance_id}}:
         "addresses": {{list of addresses}} 
         "encap_type": {{encap type}}
         "vni": {{vni}}
+        "region_id": {{local region id}}
```

Comment:
Region ID is already added to the APPLIANCE table. This is the wrong deprecated Routing Appliance table

Reply:
Thanks for catching. addressed